### PR TITLE
[quant][be] Reduce warnings in tests

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #108925
* #108924
* #108923
* __->__ #108922
* #108921
* #108920

Summary:
att

Test Plan:
python test/test_quantization.py TestQuantizePT2E

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

File: torch/ao/quantization/pt2e/utils.py [hunk 1]
```diff
@@ -166,13 +166,14 @@ def fold_bn_weights_into_conv_node(
     setattr(m, weight_attr_name, fused_weight)
     if conv_bias_node is not None:
         bias_attr_name = conv_bias_node.target
+        setattr(m, bias_attr_name, fused_bias)  # type: ignore[arg-type]
     else:
         bias_attr_name = weight_attr_name + "_bias"
+        setattr(m, bias_attr_name, fused_bias)  # type: ignore[arg-type]
```
Comment (andrewor14) [inline torch/ao/quantization/pt2e/utils.py:172]: Is this change related?

**Review by andrewor14 — APPROVED** [review]

File: torch/ao/quantization/pt2e/utils.py (hunk 1 above)
Comment (jerryzh168) [reply torch/ao/quantization/pt2e/utils.py:172]: yeah this is removing one of the warnings
